### DISCUSSION TITLE
Add Language

### DISCUSSION THREAD

**Discussion by rinrinx2** [spencerwooo/onedrive-vercel-index#587] [Ideas]
https://github.com/spencerwooo/onedrive-vercel-index/pull/582

**Comment by hirockstar** [comment]
complete the translation

**Reply by rinrinx2** [reply]
all completed.  Is there any other place to turn?
@therockstarind

**Reply by hirockstar** [reply]
check [rdrive](https://rdrive.in)

**Reply by hirockstar** [reply]
still left these translation as far i found.
"Email": "Email",
"Grid": "Grid",
"List": "List",

**Reply by rinrinx2** [reply]
I fixed the errors.  it's over now

**Reply by hirockstar** [reply]
where is new (jason) file???
